aws-gitlab-main.yml

```diff
@@ -290,6 +313,38 @@ Resources:
         - !Ref GitLabPublicSubnetA
         - !Ref GitLabPublicSubnetB
 
+  #RDS POSTGRESQL DATABASE
+  GitLabDBSubnetGroup: 
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: Group includes GitLabPrivateSubnetA and GitLabPrivateSubnetB
+      #DBSubnetGroupName: #may not need
+      SubnetIds:
+        - !Ref GitLabPrivateSubnetA
+        - !Ref GitLabPrivateSubnetB 
+
+  GitLabDB:
+    Type: AWS::RDS::DBInstance 
+    Properties: 
+      AllocatedStorage: !Ref GitLabAllocatedStorageParameter #provisioned IOPS (SSD) for storage type?
+      AutoMinorVersionUpgrade: False
+      #BackupRetentionPeriod:     #preferred backup settings == ? 
+      #DBClusterIdentifier: #related to Redis Cluster later?
+      DBInstanceClass: !Ref DBInstanceClassParameter 
+      DBInstanceIdentifier: !Ref DBInstanceIdentifierParameter
+      DBSecurityGroups: 
+        - !Ref GitLabRDSSecurityGroup
+      DBSubnetGroupName: !Ref GitLabDBSubnetGroup 
+      Engine: postgres #or paramaterize? 
+      MasterUsername: !Ref GitLabDBMasterUsernameParameter
+      MasterUserPassword: !Ref GitLabDBMasterUserPasswordParameter 
+      MultiAZ: True
+      #PreferredBackupWindow: 
+      #PreferredMaintenanceWindow: 
+
+      #it asks to make a 'standby instance'?? how
```

Comment:
Appears this can only be done via boto3:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Client.create_db_instance
Multi-AZ: True setting
Ignore for cloudformation, will do later